docs/tables/mongodbatlas_cluster.md

```diff
@@ -0,0 +1,67 @@
+# Table: mongodbatlas_cluster
+
+MongoDB Atlas cluster is a NoSQL Database-as-a-Service offering in the public cloud (available in Microsoft Azure, Google Cloud Platform, Amazon Web Services).
+
+## Examples
+
+### Basic Info
+
+```sql
+select
+  id,
+  name,
+  project_id
+from
+  mongodbatlas_cluster
+```
+
+### Get auto scaling details of all clusters
+
+```sql
+select
+  id,
+  name,
+  auto_scaling['autoIndexingEnabled'] as autoscaling_autoindexingenabled,
```

Comment:
What's the difference between using this type of subscripting vs. using `->` and `->>`? Does Postgres 14 recommend one way over the other, or are they for different use cases?

I know that `auto_scaling -> 'autoIndexingEnabled'` would return json/jsonb, but `auto_scaling ->> 'autoIndexingEnabled'` returns a text type value. Is there something similar with the `auto_scaling['audoIndexingEnabled']` syntax where we could get a text value over a json/jsonb value?